contracts/hyphen/LiquidityFarming.sol

```diff
@@ -381,6 +383,40 @@ contract HyphenLiquidityFarming is
         return rewardRateLog[_baseToken][rewardRateLog[_baseToken].length - 1].rewardsPerSecond;
     }
 
+    function migrateNftsToV2(uint256[] calldata _nftIds, IHyphenLiquidityFarmingV2 _farmingV2) external {
+        uint256 length = _nftIds.length;
+        address payable msgSender = payable(_msgSender());
+
+        for (uint256 i = 0; i < length; ) {
+            uint256 nftId = _nftIds[i];
+            uint256 index = getStakedNftIndex(_msgSender(), nftId);
+
+            require(nftIdsStaked[msgSender][index] == nftId, "ERR__NOT_OWNER");
+            require(nftInfo[nftId].staker == msgSender, "ERR__NOT_OWNER");
+
+            nftIdsStaked[msgSender][index] = nftIdsStaked[msgSender][nftIdsStaked[msgSender].length - 1];
+            nftIdsStaked[msgSender].pop();
```

Comment:
What is being done here?

Reply:
We are deleting the element at index `index` by copying the last element in the array to the position `index` and then reducing the length of the array by 1

Reply:
Got it.. i thought the loop is also applied on same array.